doc/manual/source_localization/forward.rst

```diff
@@ -520,7 +373,7 @@ The columns of the tables contain the following data:
   of the baseline of the gradiometer to show that the output is in
   T/m.
 
-.. note:: The coil geometry information is stored in the file $MNE_ROOT/share/mne/coil_def.dat, which is automatically created by the utility mne_list_coil_def , see :ref:`BJEHHJIJ`.
+.. note:: The coil geometry information is stored in the file $MNE_ROOT/share/mne/coil_def.dat
 
 .. XXX : table of normal coil description is missing
 
```

Comment:
If its the case that we did not keep the tessellation, shall we keep these ones knowing that we are missing 4/5 in [this plot_meg_sensors example](https://martinos.org/mne/dev/auto_examples/visualization/plot_meg_sensors.html)

Reply:
Then (thinking out loud)

1 - why we only have neuromag info and not the other 4?
2 - If it's important for its computation 
https://mne-tools.github.io/stable/auto_tutorials/plot_forward.html#compute-forward-solution why is not a parameter here https://mne-tools.github.io/stable/generated/mne.make_forward_solution.html#mne.make_forward_solution ? Then the only explanation is that this is embedded in the  `-trans.fif`, in the BEM or in the raw file (which is what this example suggest https://martinos.org/mne/dev/auto_examples/visualization/plot_meg_sensors.html) and by the end, it's just a field in `raw.info['chs'][_]['coil_type']`

https://github.com/mne-tools/mne-python/blob/7950ab72a5df367cf311392a4596a2a395c9da1c/mne/channels/channels.py#L1318

so the 3 of them should be linked together: forward (background), forward (tutorial), and coil plotting (example). _PLUS nothing should be said in the cookbook_

---

This also shows that maybe the order of `plot_forward.py` might not be correct. Not only 'cos it is inconsistent with the rest of the doc (trans, source, bem) vs (trans, bem, source), but also 'cos the coil has to go somewhere.